x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/ilm/IndexLifecycleTransition.java

```diff
@@ -246,13 +247,30 @@ static ClusterState moveClusterStateToPreviouslyFailedStep(
 
             LifecyclePolicyMetadata policyMetadata = ilmMeta.getPolicyMetadatas()
                 .get(LifecycleSettings.LIFECYCLE_NAME_SETTING.get(indexMetadata.getSettings()));
-            LifecycleExecutionState nextStepState = IndexLifecycleTransition.updateExecutionStateToStep(
-                policyMetadata,
-                lifecycleState,
-                nextStepKey,
-                nowSupplier,
-                true
-            );
+
```

Comment:
If the only difference between the two branches of the if/else is the boolean that's being passed through, would it be more readable to extract that as a variable and then pass the variable (with no if/else)?